content/2017/1/11/typography.rst

```diff
@@ -25,6 +25,10 @@ after:
 How to Choose Engaging & Accessible Typography for Your Website Brand
 =====================================================================
 
+.. callmacro:: blog/macros.j2#byline
+   get_blog_entries_by_bird('kit')[0]
```

Comment:
`option_spec` was the motivator, yes. I'll try the `defaultdict` approach, but there's a nonzero chance that it uses `.keys()` in which case we'll still have trouble. I'll hack at it, though.